src/AzureRenderManager/WebApp/Views/Environments/View/Manager.cshtml

```diff
@@ -244,6 +266,15 @@
         $(document).ready(function () {
             registerCheckboxEnabledFormSection("JoinDomain", "DomainForm");
             registerCheckboxEnabledFormSection("DeadlineEnvironment_RunAsService", "DeadlineServiceForm");
+
+            if ('@Model.Error' !== '')
```

Comment:
you could do this in the C# layer so the code is never emitted if it's empty

Reply:
Good idea, will do it another time though as there's lots of these.